src/NodeDocumentationMarkdownGenerator/MdFileInfo.cs

```diff
@@ -0,0 +1,121 @@
+﻿using System;
+using System.Xml;
+using Dynamo.Graph.Workspaces;
+using Dynamo.Logging;
+using Dynamo.Search.SearchElements;
+
+namespace NodeDocumentationMarkdownGenerator
+{
+    internal class MdFileInfo
+    {
+        /// <summary>
+        /// Name of the node
+        /// </summary>
+        public string NodeName { get; }
+
+        /// <summary>
+        /// Nodes namespace, this is used to generate the md file name
+        /// </summary>
+        public string NodeNamespace { get; }
+
+        public string FullCategory { get; }
+
+        public string QualifiedFileName { get; }
+
+        /// <summary>
+        /// Name of external library this node belongs to
+        /// </summary>
+        public string ExternalLib { get; }
```

Comment:
a few extra chars makes it clearer
```suggestion
        public string ExternalLibraryName { get; }
```